api/config/config_local.json

```diff
@@ -45,15 +45,17 @@
     "operationalTerraWorkspaceNamespace": "aou-wgs-cohort-extraction",
     "operationalTerraWorkspaceName": "aouwgscohortextraction",
     "operationalTerraWorkspaceBucket": "fc-56d2f6f5-3efa-46f7-8c01-0911fd77f888",
-    "extractionMethodConfigurationNamespace": "aouwgscohortextraction-test",
-    "extractionMethodConfigurationName": "GvsExtractCohortFromSampleNames",
-    "extractionMethodConfigurationVersion": 15,
-    "extractionMethodLogicalVersion": 3,
     "extractionDestinationDataset": "fc-aou-cdr-synth-test-2.wgs_extracted_cohorts",
-    "minExtractionScatterTasks": 20,
-    "extractionScatterTasksPerSample": 4,
     "gatkJarUri": "gs:\/\/all-of-us-workbench-test-genomics\/wgs\/gatk-package-4.2.0.0-533-g9c2aa67-SNAPSHOT-local.jar",
-    "enableJiraTicketingOnFailure": false
+    "enableJiraTicketingOnFailure": false,
+    "cdrv7": {
+      "methodNamespace": "aouwgscohortextraction-test",
+      "methodName": "GvsExtractCohortFromSampleNames",
+      "methodRepoVersion": 15,
+      "methodLogicalVersion": 3,
+      "minExtractionScatterTasks": 20,
+      "extractionScatterTasksPerSample": 4
+    }
   },
   "cdr": {
```

Comment:
Now when `cdrv8` is missing? Any NPE when loading that config?

Reply:
Tested locally - this works.  `cdrv8` exists but its fields do not.  There are no CDRs yet which cause the code to reference these fields, so no NPEs.